src/apic-extension/azext_apic_extension/custom.py

```diff
@@ -113,12 +119,16 @@ def _output(self, *args, **kwargs):
 
         if result:
             response_format = result['format']
-            exportedResults = result['value']
+            value = json.loads(result['value'])
```

Comment:
if `response-format` is `link`, the value will be a url instead of a json string

Reply:
fixed bad logic